spec/date.html

```diff
@@ -414,9 +415,10 @@ <h1>CreateDate ( _y_, _m_, _d_ [ , _newTarget_ ] )</h1>
     </emu-clause>
 
     <emu-clause id="sec-temporal-datefromstring" aoid="DateFromString">
-      <h1>DateFromString ( _date_ )</h1>
+      <h1>DateFromString ( _date_, _disambiguation_ )</h1>
```

Comment:
Hmm, did we discuss this? I'm not sure if we want to support anything besides "reject" here.

Reply:
See https://github.com/tc39/proposal-temporal/issues/260#issuecomment-591711184, this is an open question.